content/src/content/jcr_root/apps/core/wcm/components/page/v2/page/footer.html

```diff
@@ -19,5 +19,7 @@
          data-sly-call="${clientlib.js @ categories=clientlibCategories}"></sly>
     <sly data-sly-include="customfooterlibs.html"></sly>
     <sly data-sly-resource="${'cloudservices' @ resourceType='cq/cloudserviceconfigs/components/servicecomponents'}"></sly>
-    <sly data-sly-resource="${'cloudconfig-footer' @ resourceType='cq/cloudconfig/components/scripttags/footer'}"></sly>
+    <sly data-sly-use.pip="com.adobe.granite.license.ProductInfoProvider" data-sly-test.aemVersion="${pip.productInfo.version}"
```

Comment:
Do not have this logic in the HTL script, but rather move it to a dedicated model class.

Reply:
I considered doing that but adding a `Page#hasCloudconfigSupport` method seemed clunky and misleading (as it's a product feature, not a page property). @joerghoh, would that make better sense in your opinion?

Reply:
I would favor the approach of having the logic in the Page model.

Reply:
Like @vladbailescu said, I find the approach brittle given how often the cloud configs are changed. Having something like that in the `Page` model leads to API changes, whereas doing this in the `headlibs.html` + `footer.html` is clean enough.

Reply:
Do not change the Page API (it's already too large), but rather  move this logic into a sling model (ProductFeatureModel).

Please consider that the way the core components are built should be following best practices; and doing a version check in HTL scripts is definitly not.

Reply:
Again, providing an API that describes the product features from the Core Components seems out-of-order, we already make use of the `com.adobe.granite.license.ProductInfoProvider`. 

Unfortunately the version check is a bit verbose in HTL but apart from this I don't really see how checking the WCM mode or a product feature is preferable to a version check.

Reply:
My point is that this check is done at the wrong place (no matter how it is implemented, that's a different question). If this check would be in a Java Model, it would be much easier to come up with a different way to determining this information.

Reply:
How about extending the Page model (not the Page API) with a method `hasCloudconfigSupport`, as suggested by @vladbailescu ?

Reply:
I am fine with it

Reply:
@joerghoh @jckautzmann Please check 017eccc. For me the only upside is that we can unit-test the version comparison.